iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/planner/plan/node/ExplainAnalyzeNode.java

```diff
@@ -74,6 +68,8 @@ public PlanNode replaceChildren(List<PlanNode> newChildren) {
   protected void serializeAttributes(ByteBuffer byteBuffer) {
     PlanNodeType.EXPLAIN_ANALYZE.serialize(byteBuffer);
     ReadWriteIOUtils.write(verbose, byteBuffer);
+    ReadWriteIOUtils.write(queryId, byteBuffer);
+    ReadWriteIOUtils.write(timeout, byteBuffer);
```

Comment:
throw exception for these serde methods and add comments for that.

Reply:
done